rpa-manager/modules/ROOT/pages/botmanagement-know.adoc

```diff
@@ -67,7 +67,7 @@ The bot should not be used in the production phase.
 The bot can be used in the production phase.
 
 |*Timezone*
-|The timezone the bot's installation computer uses. The bot will interpret all times where no time zone is assigned to be set in its own time zone.
+|The timezone the bot's installation computer uses. The bot interprets all times where no time zone is assigned to be set in its own time zone.
```

Comment:
```suggestion
|The timezone that the bot's installation computer uses. The bot interprets all times where no time zone is assigned to be set in its own time zone.
```

Reply:
This would make it ambiguous:
The bot interprets all times [for which no time zone is set in its own time zone]. -> wrong
vs
The bot interprets all times [for which no time zone is set] in its own time zone. -> right